dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/SerializerTest.java

```diff
@@ -324,16 +324,20 @@ void createHistogramLines_Delta() {
                         NANOS_TS_2,
                         EMPTY_ATTRIBUTES,
                         125.5,
-                        null,
-                        null,
+                        false,
```

Comment:
This is an internal representation that are using only for the tests, and it had some slight changes.